src/pages/Users.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import Button from '../components/Button/Button';
+import classes from './Users.module.css';
+import User from '../components/User/User';
+import Modal from '../components/Modals/Modal';
+import { USERS_MODAL, DELETE_USER, openCreateUserModal } from '../utilities/actionCreators';
+
+export default class Users extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false,
+      selectedModal: '',
+      usersList: [],
+    };
+    this.dispatch = this.dispatch.bind(this);
+  }
+
+  componentDidMount() {
+    this.setState((prevState) => ({
+      ...prevState,
+      usersList: [
+        {
+          id: 1,
+          fullname: 'Aliaksandr Razumny',
+          direction: 'React',
+          education: 'BNTU',
+          start: '01.04.2021',
+          age: 25,
+        },
+        {
+          id: 2,
+          fullname: 'Joe Baiden',
+          direction: '.NET',
+          education: 'University of Pennsylvania',
+          start: '25.03.2021',
+          age: 78,
+        },
+        {
+          id: 3,
+          fullname: 'Vladimir Putin',
+          direction: 'Java',
+          education: 'LGU',
+          start: '15.04.2021',
+          age: 68,
+        },
+        {
+          id: 4,
+          fullname: 'Angela Merkel',
+          direction: 'Salesforce',
+          education: 'Hamburg Univercity',
+          start: '12.04.2021',
+          age: 66,
+        },
+        {
+          id: 5,
+          fullname: 'Barack Obama',
+          direction: 'PHP',
+          education: 'Harvard Univercity',
+          start: '29.03.2021',
+          age: 59,
+        },
+        {
+          id: 6,
+          fullname: 'Xí Jìnpíng',
+          direction: 'React',
+          education: 'Beijin Univercity',
+          start: '10.04.2021',
+          age: 67,
+        },
+        {
+          id: 7,
+          fullname: 'Emmanuel Makron',
+          direction: '.NET',
+          education: 'Paris Univercity',
+          start: '05.04.2021',
+          age: 43,
+        },
+      ],
+    }));
+  }
+
+  dispatch(action) {
+    switch (action.type) {
+      case DELETE_USER:
+        this.setState((prevState) => ({
+          ...prevState,
+          usersList: prevState.usersList.filter((item) => item.id !== action.selectedID),
+        }));
+        break;
+      case USERS_MODAL:
+        this.setState((prevState) => ({
+          ...prevState,
+          isOpen: !prevState.isOpen,
+          selectedModal: action.modalType,
```

Comment:
if you want to use dispatch what the sense to use the same code in several components ? It doesn't apply any benefit

Reply:
Good advice, but difficult for solve! Right? 
```suggestion
switch (action.type) {
    case USER_MODAL:
    case TASK_MODAL:
      state = {
        ...prevState,
        isOpen: !prevState.isOpen,
        selectedModal: action.modalType,
      };
      return state;
    default:
      return prevState;
 }
```

Reply:
Good advice, but difficult for solve! Right? 
```suggestion
  switch (action.type) {
    case USER_MODAL:
    case TASK_MODAL:
    case USERS_MODAL:
      state = {
        ...prevState,
        isOpen: !prevState.isOpen,
        selectedModal: action.modalType,
      };

      return state;
    default:
      return prevState;
  }
```